content/posts/tekton-jx-pipelines.md

```diff
@@ -0,0 +1,954 @@
+---
+author:
+  name: "David Cañadillas"
+title: "Jenkins X Orchestration: More than Tekton on Steroids"
+date: 2019-07-20T19:39:10+01:00
```

Comment:
I would just update the date to today so your post is at the top of the list. Otherwise looks good!

Reply:
So true!